From 2d0687b5ba9f17981d67e931fc823321051e52d7 Mon Sep 17 00:00:00 2001
From: DarkRain <darkrainx@users.sourceforge.net>
Date: Sat, 18 Oct 2014 08:49:57 -0500
Subject: [PATCH 21/40] * Set g_endlessaliens to 0 on skirmish shutdown.

* Turns out that it can't be deleted after a battle due to it acquiring flags.

Conflicts:
	src/client/cgame/skirmish/cl_game_skirmish.cpp
---
 src/client/cgame/skirmish/cl_game_skirmish.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/client/cgame/skirmish/cl_game_skirmish.cpp b/src/client/cgame/skirmish/cl_game_skirmish.cpp
index d0e4fd009b..21d28a017b 100644
--- a/src/client/cgame/skirmish/cl_game_skirmish.cpp
+++ b/src/client/cgame/skirmish/cl_game_skirmish.cpp
@@ -341,6 +341,8 @@ static void GAME_SK_Shutdown (void)
 	cgi->Cmd_RemoveCommand("sk_prevequip");
 	cgi->Cmd_RemoveCommand("sk_initequip");
 	cgi->Cmd_RemoveCommand("game_go");
+	/* You really don't want this with campaign */
+	cgi->Cvar_ForceSet("g_endlessaliens", "0");
 
 	cgi->UI_ResetData(OPTION_DROPSHIPS);
 	cgi->UI_ResetData(OPTION_UFOS);
-- 
2.16.1

